frontend-cert/js-projects/lab-madlibs/user-stories.md

```diff
@@ -0,0 +1,15 @@
+1. You should define the following variables using `let`:
+- `adjective`
+- `noun`
+- `verb`
+- `place`
+- `adjective2`
+- `noun2`
+
+1. You should assign the above variables some values.
+
+1. You should use the following story template to create a story:
+
+`"'Once upon a time, there was a(n) [adjective] [noun] who loved to eat [noun2]. The [noun] lived in a [place] and had [adjective2] nostrils that blew fire when it was [verb]."'"`;
```

Comment:
```suggestion
`"Once upon a time, there was a(n) [adjective] [noun] who loved to eat [noun2]. The [noun] lived in a [place] and had [adjective2] nostrils that blew fire when it was [verb]."`;
```
Not sure if single quotes were there for a reason.

Reply:
These quotes shouldn't have been be there. Fixed it.